modules/aerospike/aerospike.go

```diff
@@ -0,0 +1,71 @@
+package aerospike
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	port          = "3000/tcp"
+	containerName = "tc_dynamodb_local"
+)
+
+type AerospikeContainer struct {
+	testcontainers.Container
+	Host string
+	Port int
+}
+
+// Run creates an instance of the Aerospike container type
+func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustomizer) (*AerospikeContainer, error) {
+	req := testcontainers.ContainerRequest{
+		Image:        img,
+		ExposedPorts: []string{"3000/tcp", "3001/tcp", "3002/tcp", "3003/tcp"},
```

Comment:
suggestion: we can probably extract the ports as constants so that we can reuse the values in the wait strategies. Thoughts?